### ISSUE TITLE
one more protein to look at

### ISSUE THREAD

**Issue by timlevine** [UCL/HHyeast-server#6]
Urn1 = YPR152C : does the server show the 2nd WW domain in residues 50-90?

**Comment by ilectra** [comment]
Do you mean this?
![ypr152c](https://user-images.githubusercontent.com/22891967/32945121-1e10e462-cb8a-11e7-810f-64471b12038b.png)

**Comment by timlevine** [comment]
yes - and it's "perfect" having the hits between residues 55 and 90 (which is missing elsewhere)
Tim

**Comment by ilectra** [comment]
- [ ] Run checks for the existing files, using new scripts.
- [ ] Get the still missing files done with the re-processing

**Comment by ilectra** [comment]
@timlevine The list of proteins we don't have results for:

YPLCTy4-1.0.ssw11.hhr
YDR406W.0.ssw11.hhr
YGR199W.0.ssw11.hhr
Q0050.0.ssw11.hhr
YGR003W.0.ssw11.hhr
YAR062W.0.ssw11.hhr
YCL011C.0.ssw11.hhr
YNR047W.0.ssw11.hhr
YKL203C.0.ssw11.hhr
YJL207C.0.ssw11.hhr
YOR116C.0.ssw11.hhr
YBL099W.0.ssw11.hhr
YDR145W.0.ssw11.hhr
YOR393W.0.ssw11.hhr
YBL088C.0.ssw11.hhr
YDR422C.0.ssw11.hhr
YDR484W.0.ssw11.hhr
YMR284W.0.ssw11.hhr
YNL163C.0.ssw11.hhr
YBR276C.0.ssw11.hhr
YGR204W.0.ssw11.hhr
YPL249C.0.ssw11.hhr
YDR416W.0.ssw11.hhr
YKL164C.0.ssw11.hhr
YOR343W-B.0.ssw11.hhr
YLR096W.0.ssw11.hhr
YOR251C.0.ssw11.hhr
YIL092W.0.ssw11.hhr
YOR162C.0.ssw11.hhr
YMR013C.0.ssw11.hhr
YGL097W.0.ssw11.hhr
YAL026C.0.ssw11.hhr
YLL001W.0.ssw11.hhr
YIL147C.0.ssw11.hhr
YAL008W.0.ssw11.hhr
YGL062W.0.ssw11.hhr
YGR240C.0.ssw11.hhr
YKL209C.0.ssw11.hhr
YKL188C.0.ssw11.hhr
YPL226W.0.ssw11.hhr
YGR281W.0.ssw11.hhr
YBR084W.0.ssw11.hhr
YLR398C.0.ssw11.hhr
YLR163C.0.ssw11.hhr
YPL137C.0.ssw11.hhr
YER132C.0.ssw11.hhr
YMR176W.0.ssw11.hhr
YGR061C.0.ssw11.hhr
YGL100W.0.ssw11.hhr
YMR296C.0.ssw11.hhr
YER164W.0.ssw11.hhr
YOL152W.0.ssw11.hhr
YPR009W.0.ssw11.hhr
YGR245C.0.ssw11.hhr
YFL007W.0.ssw11.hhr
YDR359C.0.ssw11.hhr
YEL060C.0.ssw11.hhr
YBR225W.0.ssw11.hhr
YDL040C.0.ssw11.hhr
YJR066W.0.ssw11.hhr
YFL057C.0.ssw11.hhr
YOL021C.0.ssw11.hhr
YPR032W.0.ssw11.hhr
YGL197W.0.ssw11.hhr
YNR033W.0.ssw11.hhr
YGL086W.0.ssw11.hhr
YAR019C.0.ssw11.hhr
YOL123W.0.ssw11.hhr

**Comment by timlevine** [comment]
Hi Ilektra

I repeated the search online with Pdr15 (Yrd406w) and it worked (see https://toolkit.tuebingen.mpg.de/#/jobs/pdr15).


There are 67 yeast genes in the list - I can't make out the top entry (YPLCTy4-1)

Their average number of annotated domains in SGD is 9.1 compared to 4.4 for the 6717 in a list I found (this must contain the dubious ORFs - if you have an updated list excluding those, please send!). This increase in domain number is statistically significant (p=1e-10), so on the whole these proteins are more complicated than average.

There is little difference between the file of hits generated for one I tried and what a "sister" protein (Pdr10) generated, so it seems unlikely that there is something about the data flow/file size being a bit too big.

Rather than go through the other 66, we should decide if you can run them again (only 1%) or if you want me to run them online and send you the files!!!!!

All the best,
Tim


________________________________

**Comment by timlevine** [comment]
Thanks for that.

Will HHyeast have a simple URL by December 17?
